crates/bevy_ecs/src/schedule/label.rs

```diff
@@ -4,24 +4,40 @@ use bevy_utils::define_label;
 define_label!(
     /// A strongly-typed class of labels used to identify [`Stage`](crate::schedule::Stage)s.
```

Comment:
IMO these docs are the actually user-facing ones; the ignore fields trick should be discussed here too.

Reply:
At least in rust-analyzer, [these docs](https://github.com/bevyengine/bevy/pull/5715/files/dfb3841907e1f6c23846b610fbe4fed66d026427#diff-37689ed3f6a374cc5d67fccdaf4258f06ab7b694c25397b588c336c4beae96d4) are the ones you see when hovering over the derive macro, so I think it makes more sense to keep that information there.

Reply:
Ah right. Okay, I'm fine with that then.